vercel.json

```diff
@@ -1,15 +1,5 @@
 {
   "rewrites": [
-    {
-      "source": "api/:path*",
-      "destination": "api/:path",
-      "has": [
-        {
-          "type": "host",
-          "value": "gloss-translation-api.*"
-        }
-      ]
-    },
```

Comment:
This rewrite doesn't turn out to work for nested paths on the api, so I had to refactor these rewrites again